pelicanconf.py

```diff
@@ -60,7 +60,7 @@
               'Software, Open Source, and related topics.'
 
 LINKS = (
-    ('IRC', 'https://webchat.freenode.net/?channels=nairobilug'),
+    ('IRC', 'https://web.libera.chat/?channels=nairobilug'),
```

Comment:
ditto https://github.com/nairobilug/nairobilug.or.ke/pull/194/files#r655467714

Reply:
Resolved in 51c6ba50ec0f7dfa43793cd5218065066369733f.